Tools/Scripts/libraries/webkitscmpy/webkitscmpy/remote/git_hub.py

```diff
@@ -362,17 +362,22 @@ def request(self, path=None, params=None, headers=None, authenticated=None, pagi
                 name=self.name,
                 path='/{}'.format(path) if path else '',
             )
-        response = self.session.request(method, url, params=params, json=json, headers=headers, auth=auth, files=files, data=data)
+        response = self.session.request(method, url, params=params, json=json, headers=headers, auth=auth, files=files, data=data, stream=stream)
+        is_json_response = 'application/json' in response.headers.get('Content-Type', '')
```

Comment:
I think it should just be `json` in `Content-Type`, right? `text/json` would also be valid.

Reply:
(and also what our mock servers are responding with, I think)

Reply:
I’ll make it more resilient. The server actually returns` application/json`

Reply:
Problem solved.